resources/build_essential.rb

```diff
@@ -27,6 +27,9 @@
   when 'debian'
     package %w( autoconf binutils-doc bison build-essential flex gettext ncurses-dev )
   when 'amazon', 'fedora', 'rhel'
+    file '/etc/resolv.conf' do
+      content 'nameserver 8.8.8.8'
+    end if node['platform_family'] == 'amazon'
```

Comment:
Would an `only_if` be more appropriate here?

Reply:
I'm not sure if this is the best way to fix it - I think this was fixed here: https://github.com/someara/dokken-images/pull/14 we just need to consume it?